.github/workflows/ci.yml

```diff
@@ -14,18 +14,12 @@ env:
   cache-version: v1
 
 jobs:
-  gradle-wrapper-validation:
-    name: gradle-wrapper-validation
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: gradle/wrapper-validation-action@v1
   build:
     runs-on: ubuntu-latest
     timeout-minutes: 20
     steps:
       - name: Check out repository code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
`test:` job's checkout is still v3 I think, see warnings:
https://github.com/android/android-test/actions/runs/7702474927?pr=2129